Source/Languages/Casing/CamelCaseConverter.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="CaseStyleConverter.ts" />
+
+namespace GLS.Languages.Casing {
+    "use strict";
+
+    /**
+     * Converts a name to camelCase.
+     */
+    export class CamelCaseConverter extends CaseStyleConverter {
+        /**
+         * Converts a name to camelCase.
+         * 
+         * @param name   A name to convert.
+         * @returns The name's equivalent in camelCase.
+         */
+        public convert(name: string): string {
+            name = super.convert(name);
+
+            return name[0].toLowerCase() + name.substring(1);
```

Comment:
Precisely.

The only difference between camelCase and PascalCase is the first letter of the first word.